PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskRecursiveSoftDrop.h

```diff
@@ -0,0 +1,81 @@
+#ifndef ALIANALYSISTASKRECURSIVESOFTDROP_H
+#define ALIANALYSISTASKRECURSIVESOFTDROP_H
+
+class TH1;
+class TH2;
+class TH3;
+class TH3F;
+class TTree;
+class THnSparse;
+class TClonesArray;
+class TArrayI;
+class AliAnalysisManager;
+class AliJetContainer;
+class AliEmcalJetFinder;
+class AliFJWrapper;
+
+#include "AliAnalysisTaskEmcalJet.h"
+#include "AliFJWrapper.h"
+#include "AliClusterContainer.h"
+#include "FJ_includes.h"
+
+const Int_t nBranches = 4;
```

Comment:
This should be inside of your class - otherwise I think you are adding some new constant to the global namespace. I would suggest creating an enum for it in your class

Reply:
And since we are already at that step please follow the naming convention for constants in order to not mix between local variables, global variables, class members and constants.